<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GuiAddBook</class>
 <widget class="QMainWindow" name="GuiAddBook">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>492</width>
    <height>284</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="lInformationBook">
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Book information</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="3" column="1">
       <widget class="QLineEdit" name="inputPublisher"/>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="inputAuthor"/>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="lTitle">
        <property name="text">
         <string>Title:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="inputCategory"/>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="inputTitle"/>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="lYear">
        <property name="text">
         <string>Year:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="inputYear"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="lCategory">
        <property name="text">
         <string>Catagory:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lAuthor">
        <property name="text">
         <string>Author:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="lPublisher">
        <property name="text">
         <string>Publisher:</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="lExit">
        <property name="text">
         <string>Exist:</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLineEdit" name="inputExist"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="bAdd">
        <property name="text">
         <string>Add</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="bCancel">
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>inputTitle</tabstop>
  <tabstop>inputAuthor</tabstop>
  <tabstop>inputCategory</tabstop>
  <tabstop>inputPublisher</tabstop>
  <tabstop>inputYear</tabstop>
  <tabstop>bAdd</tabstop>
  <tabstop>bCancel</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
